includes/class-compatibility-service.php

```diff
@@ -47,11 +47,14 @@ public function init_hooks() {
 	 * @return void
 	 */
 	public function update_compatibility_data() {
+		$active_plugins = get_option( 'active_plugins' );
```

Comment:
It should never happen, but `get_option` can return `false`. We should add a check and if `$active_plugins` is not an array, it should be changed to an empty array to match what's expected to be received at the server. 

A test would be needed to make sure that's working correctly, as well.

Reply:
Resolving this per our discussion in slack: p1704818339032419-slack-C01B8KNUYSW